web/rest/client/features/provider/conferenceRoom/postConferenceRoom.feature

```diff
@@ -14,7 +14,8 @@ Feature: Create conference rooms
           "name": "newConferenceRoom",
           "pinProtected": true,
           "pinCode": "1234",
-          "maxMembers": 1
+          "maxMembers": 1,
+          "announceUserCount": "first"
```

Comment:
There should be some changes in conferenceRoom/getConferenceRoom.feature as well

Reply:
Updated the Dto and Get tests :+1: